_data/students.yml

```diff
@@ -29,3 +29,6 @@
   github: STEdirisuriya
   image: supun.png
 
+- name: Hussain Arif
+  github: HussainArif12
+  image: nick photo.png
```

Comment:
The name of the image should be more informative, make it about the person whose image it is.